randovania/games/prime1/json_data/Phazon Mines.txt

```diff
@@ -1946,15 +1948,25 @@ Extra - aabb: [90.076614, 160.53384, -176.2045, 166.0206, 330.8736, -124.46773]
   * Extra - dock_index: 1
   * Extra - nonstandard: False
   > Door to Phazon Mining Tunnel
-      Any of the following:
-          All of the following:
-              Space Jump Boots
-              R-Jump (Intermediate) or Use Grapple Beam
-          All of the following:
-              Scan Visor
-              Any of the following:
-                  Combat/Scan Dash (Hypermode)
-                  Combat/Scan Dash (Expert) and Use Grapple Beam
+      All of the following:
+          Any of the following:
+              All of the following:
+                  Space Jump Boots
+                  Any of the following:
+                      R-Jump (Intermediate) or Use Grapple Beam
+                      Scan Visor and Combat/Scan Dash (Intermediate)
+              All of the following:
+                  Scan Visor
+                  Any of the following:
+                      Combat/Scan Dash (Hypermode)
+                      Combat/Scan Dash (Expert) and Use Grapple Beam
+          Any of the following:
```

Comment:
Uncharged plasma feels pretty powerful here. At the end of the day, writing logic like this just plain sucks, so don't feel like you have to for every room.

Also remember that you are picking health values that come before the normal 1.5x scaling randovania does by default.

Reply:
You increased the numbers but my concern was that they were too high. How did you test these numbers? They should represent the best possible scenario, not the least.